scripts/validation-utils.js

```diff
@@ -33,90 +33,76 @@ module.exports = {
-      console.log(`You must set environment variable 'CIRCLECI_BUILD'.`);
-      console.log(
+      logger.error(`You must set environment variable 'CIRCLECI_BUILD'.`);
+      logger.info(
         `To set: 'export CIRCLECI_BUILD=build_num'. Where build_num contains the artifacts that will be released.`
       );
       process.exit(-1);
     }
   },
 
   checkEnvironmentVariables: () => {
-    console.log('\nVerifying environment variables have been set.');
+    logger.header('\nVerifying environment variables have been set.');
     module.exports.checkVSCodeVersion();
     module.exports.checkCircleCiToken();
     module.exports.checkCircleCiBuild();
   },
 
   checkNodeVersion: () => {
-    console.log('\nVerifying node version ' + NODE_VERSION + ' is installed.');
+    logger.header('\nVerifying node version ' + NODE_VERSION + ' is installed.');
     const [version, major, minor, patch] = process.version.match(
       /^v(\d+)\.?(\d+)\.?(\*|\d+)$/
     );
     if (
       parseInt(major) != NODE_VERSION.split('.')[0] ||
       parseInt(minor) < NODE_VERSION.split('.')[1]
     ) {
-      console.log(
+      logger.error(
         'Please update from node version ' +
-          process.version +
-          ' to ' +
-          NODE_VERSION
+        process.version +
+        ' to ' +
+        NODE_VERSION
       );
       process.exit(-1);
     }
   },
 
   checkLernaInstall: () => {
-    console.log(
-      `\nVerifying lerna is installed for node version ${NODE_VERSION}.`
-    );
+    logger.header(`\nVerifying lerna is installed for node version ${NODE_VERSION}.`);
     if (!shell.which('lerna') || !shell.which('lerna').includes(NODE_VERSION)) {
-      console.log(
-        'Lerna not found - Installing lerna version ' + LERNA_VERSION
+      logger.info(`Lerna location: ` + shell.which('lerna'));
+      logger.info(
+        `Lerna is not installed for node version ${NODE_VERSION} - Installing lerna version ${LERNA_VERSION}`
       );
       shell.exec('npm install -g lerna@' + LERNA_VERSION);
     }
   },
 
   checkVSCEInstall: () => {
-    console.log(
-      `\nVerifying vsce is installed for node version ${NODE_VERSION}.`
-    );
+    logger.header(`\nVerifying vsce is installed for node version ${NODE_VERSION}.`);
     if (!shell.which('vsce') || !shell.which('vsce').includes(NODE_VERSION)) {
-      console.log('VSCE not found - Installing latest version.');
+      logger.info('VSCE Location: ' + shell.which('vsce'));
+      logger.info(`VSCE not found for node version ${NODE_VERSION} - Installing latest version.`);
       shell.exec('npm install -g vsce');
     }
   },
 
   checkAWSCliInstall: () => {
-    console.log(
-      `\nVerifying AWS CLI is installed for node version ${NODE_VERSION}.`
-    );
+    logger.header(`\nVerifying AWS CLI is installed for node version ${NODE_VERSION}.`);
     if (!shell.which('aws')) {
-      console.log('The AWS CLI is not installed or could not be found.');
```

Comment:
Decided to remove all of these steps. Installing via pip3 seemed much more complicated (having to modify your path, dealing with python issues). From the release last week, installing via their install gui worked just fine.